android/app/src/main/java/org/fossasia/openevent/utils/Views.java

```diff
@@ -113,6 +115,31 @@ public static void changeGlowColor(int color, NestedScrollView scrollView) {
 
     }
 
+    /*Check if the version is above Lollipop to change the edge glow color of RecyclerView
+    according to the session detail activity color*/
+    @TargetApi(LOLLIPOP)
+    public static void setEdgeGlowColorRecyclerView(final RecyclerView recyclerView, final int color) {
+
+        try {
+            final Class<?> clazz = RecyclerView.class;
+            for (final String name : new String[]{"ensureTopGlow", "ensureBottomGlow", "ensureLeftGlow", "ensureRightGlow"}) {
+                Method method = clazz.getDeclaredMethod(name);
+                method.setAccessible(true);
+                method.invoke(recyclerView);
+            }
+            for (final String name : new String[]{"mTopGlow", "mBottomGlow", "mLeftGlow", "mRightGlow"}) {
+                final Field field = clazz.getDeclaredField(name);
+                field.setAccessible(true);
+                final Object edge = field.get(recyclerView);
+                final Field fEdgeEffect = edge.getClass().getDeclaredField("mEdgeEffect");
+                fEdgeEffect.setAccessible(true);
+                ((EdgeEffect) fEdgeEffect.get(edge)).setColor(color);
+            }
+        } catch (final Exception ignored) {
```

Comment:
@iamareebjamal I am using `@TargetApi(LOLLIPOP)` above the function. Try and catch is for the error thrown by functions. I am not sure what you are trying to say.